Containerfile

```
@@ -88,6 +88,11 @@ RUN rpm-ostree install $(curl https://api.github.com/repos/loft-sh/devpod/releas
   wget https://github.com/loft-sh/devpod/releases/latest/download/devpod-linux-amd64 -O /tmp/devpod && \
   install -c -m 0755 /tmp/devpod /usr/bin
 
+# Install kns/kctx and add completions for Bash
+RUN git clone https://github.com/ahmetb/kubectx /opt/kubectx && \
+    ln -s /opt/kubectx/kubectx /usr/local/bin/kctx && \
+    ln -s /opt/kubectx/kubens /usr/local/bin/kns
```

```suggestion
    ln -s /opt/kubectx/kubectx /usr/bin/kctx && \
    ln -s /opt/kubectx/kubens /usr/bin/kns
```

In this context that directory is for you when you're consuming on your local image, so it'll fail if we try to put stuff in /usr/local at the build level.